okteto.yml

```diff
@@ -4,7 +4,12 @@ securityContext:
   capabilities:
     add:
     - SYS_PTRACE
+resources:
```

Comment:
why is this wanted in the dev environment pod?

Reply:
I had troubles debugging the app. so, I updated the resources. I will remove and lets see if we have debugging troubles in future.

Reply:
Testing the debug environment with default settings [cpu: 1, memory: 1Gi] fails during modules download.
attempt 1: The debug pod dies and deployment spins up the new pod
```
google.golang.org/protobuf/types/descriptorpb
google.golang.org/protobuf/types/known/anypb
github.com/golang/protobuf/ptypes/any
Connection lost to your development container, reconnecting...
 ✓  Persistent volume successfully attached
 ✓  Images successfully pulled
 ✓  Files synchronized
    Context:   replicated.okteto.dev
    Namespace: pavansokkenagaraj
    Name:      kotsadm
    Forward:   2353 -> 2345
 ```
 
 ```
➜  ~ k get podmetrics kotsadm-okteto-99cc857d-gz2pn -o yaml
apiVersion: metrics.k8s.io/v1beta1
containers:
- name: kotsadm
  usage:
    cpu: 1995794206n
    memory: 1732132Ki
kind: PodMetrics
metadata:
  creationTimestamp: "2022-02-15T01:40:00Z"
  name: kotsadm-okteto-99cc857d-gz2pn
  namespace: pavansokkenagaraj
  selfLink: /apis/metrics.k8s.io/v1beta1/namespaces/pavansokkenagaraj/pods/kotsadm-okteto-99cc857d-gz2pn
timestamp: "2022-02-15T01:39:40Z"
window: 30s
➜  ~
```

attempt 2: worked, but could see mod download being slow.